draft-ietf-quic-http.md

```diff
@@ -942,6 +942,11 @@ A server MAY interleave one or more PUSH_PROMISE frames (see
 PUSH_PROMISE frames are not part of the response; see {{server-push}} for more
 details.
 
+The HEADERS and PUSH_PROMISE frames might reference updates to the QPACK table
+which are sent at the same time.  While these updates are not directly part
```

Comment:
Maybe the first sentence could say something like

"The HEADERS and PUSH_PROMISE frames might referenced synchronised updates to the QPACK dynamic table."

Reply:
or "HEADERS and PUSH_PROMISE frames might reference[!] concurrent updates to the QPACK dynamic table"